src/app/routes/utils/regex/utils/index.js

```diff
@@ -74,18 +74,14 @@ const homePageServices = [
   'yoruba',
 ];
 
-const servicesWithVariants = ['serbian', 'ukchina', 'zhongwen'];
-
 export const getFrontPageRegex = services => {
   let frontPages = services;
   if (isLive()) {
     frontPages = services.filter(
       service => !homePageServices.includes(service),
     );
   } else {
-    frontPages = services.filter(service =>
-      servicesWithVariants.includes(service),
-    );
+    frontPages = [];
```

Comment:
**Note to reviewers**: It is no longer possible to render front pages on local or test environments, so we've decided to return an empty array. This does generate a [less than ideal regex (services placeholder is empty)](https://github.com/bbc/simorgh/blob/2646f2350e97727bab6c5de75af04bf6fabc1ffe/src/app/routes/utils/regex/utils/__snapshots__/index.test.js.snap#L27), but this is only temporary until we can migrate all front pages -> home pages (and then delete all references to front page).